tests/test_integrated_channels/test_sap_success_factors/test_transmitters/test_learner_data.py

```diff
@@ -223,3 +223,11 @@ def test_transmit_by_course_id_success(self):
         assert payloads[0].error_message == 'error occurred'
         assert payloads[1].status == '200'
         assert payloads[1].error_message == ''
+
+    def test_override_channel_settings(self):
+        """
+        If you add any settings to the ChannelSettingsMixin, add a test here for the common default value
+        """
+        assert learner_data.SapSuccessFactorsLearnerTransmitter(
+            self.enterprise_config
+        ).INCLUDE_GRADE_FOR_COMPLETION_AUDIT_CHECK is False
```

Comment:
is it worth it to test that a SAP transmission will skip regardless of whether the grade has changed?

Reply:
hmm I can look at that , it should be doable to test that SAP specific transmitter let me check

Reply:
added test!